release_notes/363-release-notes.adoc

```diff
@@ -0,0 +1,64 @@
+[id="363-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes version 3.63.0
+include::modules/common-attributes.adoc[]
+:context: 363-release-notes
+
+toc::[]
+
+{product-title} 3.0.62 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+//To upgrade to this release from a previous version, see the link:/docs/upgrade-stackrox/[Upgrade]section.
+
+*Release date:* July 21, 2021
+
+[id="release-tag-version-change"]
+== Release tag version change
+
+This release and the subsequent releases of {product-title} will use the updated version number convention as `major-release.minor-release.patch-release`.
+
+[id="new-features"]
+== New features
+
+[discrete]
+=== Installing using an Operator
+You can now install {product-title} on {ocp} by using an Operator. The {product-title} Operator is available on the OperatorHub. See xref:../installing/install-ocp-operator.adoc#install-ocp-operator[Installing {product-title} using the Operator] for details.
+
+[discrete]
+=== Scoped access control
+We have updated the way that {product-title} handles access control. You can now define scopes for Kubernetes resources, for example, namespaces and clusters, and assign those scopes to roles. See xref:../operating/manage-user-access/manage-role-based-access-control-3630.adoc#manage-role-based-access-control-3630[Managing role-based access control in {product-title} 3.63.0 and newer] for more information.
```

Comment:
```suggestion
The way that {product-title} handles access control has been updated. You can now define scopes for Kubernetes resources, such as namespaces and clusters, and assign those scopes to roles. See xref:../operating/manage-user-access/manage-role-based-access-control-3630.adoc#manage-role-based-access-control-3630[Managing role-based access control in {product-title} 3.63.0 and newer] for more information.
```

Reply:
Also "role-based access control" could be changed to "RBAC" (depending on whether you decide to make that title change in the other files).